test/extended/testdata/gssapi/fedora/base/fedora-base-buildconfig.json

```diff
@@ -0,0 +1,47 @@
+{
```

Comment:
Just so it doesn't get forgotten, in the end all these should be YAML, or at least this is what I've heard we want to move all fixtures towards